bcfms/src/bcfms/ipa/pages/SubmitProject/steps/Step2_Details.vue

```diff
@@ -296,7 +311,12 @@ defineExpose({ isValid });
     background-color: #eaeaea !important;
     color: #000000 !important;
 }
+
 .datepicker-width {
     width: 85%;
 }
+
+.p-select-option-label {
```

Comment:
This should probably be defined in the arches common CSS so it's picked up globally. That's probably true for all the rules in this file unless there are any that actually are specific to this rule.

It's working because it's not in a scoped style tag, but it's not obvious to put it here, and we'd need to redefine these rules in subsequent workflows in this project (and in all projects) the way this is done.